docs/gpa.md

```diff
@@ -233,78 +231,23 @@ The information included in the notice of intended procurement covers all the in
-      <tr>
-        <td>X:7(b)</td>
-        <td>any conditions for participation of suppliers, including a list of information and documents that suppliers are required to submit in connection with the conditions for participation;</td>
-        <td markdown=1>
-
-* 
-</td>
-      </tr>
-      <tr>
-        <td>X:7(c)</td>
-        <td>all evaluation criteria the entity will apply in the awarding of the contract, and, except where price is the sole criterion, the relative importance of such criteria;</td>
-        <td markdown=1>
-
-* 
-</td>
-      </tr>
-      <tr>
-        <td>X:7(d)</td>
-        <td>where the procuring entity will conduct the procurement by electronic means, any authentication and encryption requirements or other requirements related to the submission of information by electronic means;</td>
+        <td>A procuring entity shall make available to suppliers tender documentation that includes all information necessary to permit suppliers to prepare and submit responsive tenders. Unless already provided in the notice of intended procurement, such documentation shall include a complete description of:</td>
         <td markdown=1>
 
-* 
-</td>
+* If at the time of publishing tender documentation (article X:7) you have already published all the information necessary for a notice of intended procurement (article VII:2), and this information has not changed, discard this article.
+* If you haven't published all the information necessary for a notice of intended procurement or if the information has changed, follow the guidance for article VII:2 to publish the new and updated data.
+        </td>
       </tr>
       <tr>
-        <td>X:7(e)</td>
-        <td>where the procuring entity will hold an electronic auction, the rules, including identification of the elements of the tender related to the evaluation criteria, on which the auction will be conducted;</td>
-        <td markdown=1>
-
-* 
-</td>
-      </tr>
-      <tr>
-        <td>X:7(f)</td>
-        <td>where there will be a public opening of tenders, the date, time and place for the opening and, where appropriate, the persons authorized to be present;</td>
+        <td>X:9</td>
+        <td>The evaluation criteria set out in the notice of intended procurement or tender documentation may include, among others, price and other cost factors, quality, technical merit, environmental characteristics and terms of delivery</td>
         <td markdown=1>
 
-* 
-</td>
-      </tr>
-      <tr>
-        <td>X:7(g)</td>
-        <td>any other terms or conditions, including terms of payment and any limitation on the means by which tenders may be submitted, such as whether on paper or by electronic means; and</td>
-        <td markdown=1>
+* Enter the *evaluation criteria set out in the notice of intended procurement or tender documentation* in `tender/awardCriteria/description`, or, if possible, split this into `AwardCriterion` objects in the `tender/awardCriteria/criteria` array.
```

Comment:
I think you mean `tender/awardCriteriaDetails`

Reply:
I meant the `awardCriteria` property from https://github.com/open-contracting-extensions/ocds_awardCriteria_extension, but it inconveniently only applies to lots :(

Reply:
Done